netbox/netbox/views.py

```diff
@@ -248,11 +251,25 @@ def get(self, request):
 
         }
 
+        new_release = None
+        new_release_url = None
+
+        if request.user.is_staff:
```

Comment:
Probably want to check for superuser status also, since superuser status oddly enough does not imply `is_staff`:

```
>>> jstretch = User.objects.get(username='jstretch')
>>> jstretch.is_superuser
True
>>> jstretch.is_staff
False
```